test/com/google/enterprise/gsafeed/AclTest.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.enterprise.gsafeed;
+
+import static org.junit.Assert.assertFalse;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.xmlunit.builder.DiffBuilder;
+import org.xmlunit.diff.Diff;
+
+/**
+ * Test Acl.
+ */
+public class AclTest {
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
```

Comment:
You're not using this anywhere. Are you planning to add failures in a followup? Do some things fail now (places where the DTD specifies alternatives, for example), while other things will only fail with our own validation (e.g., date formats, or alternatives not listed in the DTD, like feedtype for groups)?